docs/application/dotnet/guides/app-management/application-preference.md

```diff
@@ -0,0 +1,266 @@
+# Application Preferences
+
+You can manage application preferences by setting and getting them. You can also share stored preference data among applications in the same package.
+
+## Managing Application Preferences
+
+To manage preferences:
+
+- To store and retrieve variables:
+
+  To store a variable, you must create a key-value pair. Use the following function to create a key-value pair for a specific simple type:
+
+  - `Preference.Set(string key, value)`
+
+  Before storing or retrieving a variable, check whether it exists using the `Preference.Contains(string key)` function.
+
+  Use the following function to retrieve a stored variable.
+
+  - `Preference.Get<T>()`
+
+  ```csharp
+  //set/get integer value
+  Preference.Set("test_integer", 10);
+
+  if (Preference.Contains("test_integer"))
+  {
+      var testInt = Preference.Get<int>("test_integer");
+  }
+  ```
+
+- To store and retrieve string variables, use the following functions:
+
+  - `Preference.Set("test_string", "TEST VALUE")`
+  - `Preference.Get<string>("test_string")`
+
+- To list all records, use `Keys` collection:
+  - `Prefrence.Keys()`
+
+  ```csharp
+    Preference.Set("Option_enabled", true);
+    Preference.Set("active_user", "Joe");
+    Preference.Set("default_volume", 10);
+    Preference.Set("brightness", "0.6");
+
+    foreach(string key in Preference.Keys)
+    {
+        Tizen.Log.Info("TEST", string.Format("Key name: {0}", key));
+    }
+
+  ```
+
+- To safely remove record, use:
+  - `Preference.Remove(string key)`;
+
+```csharp
+    if (Preference.Contains("key"))
+    {
+        Preference.Remove("key");
+    }
+```
+
+- To remove all records, use:
+  - `Preference.RemoveAll()`
+
+## Code snippet
```

Comment:
Can this heading be rephrased or this whole section restructured, example in what order are these functions to be called?
See: https://docs1.stg.tizen.org/staging/1503/application/dotnet/guides/app-management/component-manager/#managing-running-component-context